pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -504,3 +509,12 @@ type AlertmanagerList struct {
 func (l *AlertmanagerList) DeepCopyObject() runtime.Object {
 	return l.DeepCopy()
 }
+
+// AlertmanagerTLSConfigFields defines the mTLS command line flag when starting Alertmanager.
+// +k8s:openapi-gen=true
+type ClusterTLSConfigFields struct {
+	// Server-side configuration for mutual TLS.
+	ServerTLS *WebTLSConfig `json:"server"`
```

Comment:
should we make it required?
```suggestion
        // +required
	ServerTLS WebTLSConfig `json:"server"`
```

Reply:
yep, agreed.